src/getCoinCombination.test.js

```diff
@@ -9,12 +9,46 @@ describe(`Function 'getCoinCombination':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(getCoinCombination(15)).toBeInstanceOf(Array);
   });
 
   it(`should return [0, 0, 0, 0] if amount = 0`, () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
+  });
 
+  it(`should return [1, 0, 0, 0] if amount = 1`, () => {
+    expect(getCoinCombination(1)).toEqual([1, 0, 0, 0]);
   });
 
-  // write more tests here
+  it(`should return [4, 0, 0, 0] if amount = 4`, () => {
+    expect(getCoinCombination(4)).toEqual([4, 0, 0, 0]);
+  });
+
+  it(`should return [0, 1, 0, 0] if amount = 5`, () => {
+    expect(getCoinCombination(5)).toEqual([0, 1, 0, 0]);
+  });
+
+  it(`should return [1, 1, 0, 0] if amount = 6`, () => {
+    expect(getCoinCombination(6)).toEqual([1, 1, 0, 0]);
+  });
+
+  it(`should return [0, 0, 1, 0] if amount = 10`, () => {
+    expect(getCoinCombination(10)).toEqual([0, 0, 1, 0]);
+  });
+
+  it(`should return [1, 0, 1, 0] if amount = 11`, () => {
+    expect(getCoinCombination(11)).toEqual([1, 0, 1, 0]);
+  });
+
+  it(`should return [1, 1, 1, 0] if amount = 16`, () => {
+    expect(getCoinCombination(16)).toEqual([1, 1, 1, 0]);
+  });
+
+  it(`should return [0, 0, 0, 1] if amount = 25`, () => {
+    expect(getCoinCombination(0)).toEqual([0, 0, 0, 0]);
```

Comment:
name of the test and expectation is different

Reply:
Thanks! Fixed the mistake